icu4c/source/common/unicode/uloc.h

```diff
@@ -1142,6 +1142,16 @@ uloc_forLanguageTag(const char* langtag,
                     int32_t* parsedLength,
                     UErrorCode* err);
 
+#ifndef U_HIDE_SYSTEM_API
+U_STABLE int32_t U_EXPORT2
+uloc_forLanguageTagImpl(const char* langtag,
```

Comment:
Could this go in `source\common\ulocimp.h` instead?

Reply:
^ I agree.